pkgs/development/libraries/rocwmma/default.nix

```diff
@@ -0,0 +1,157 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, writeScript
+, cmake
+, rocm-cmake
+, hip
+, openmp
+, gtest ? null
+, rocblas ? null
+, texlive ? null
+, doxygen ? null
+, sphinx ? null
+, python3Packages ? null
+, buildTests ? false
+, buildSamples ? false
+, buildDocs ? false
+, gpuTargets ? null # gpuTargets = [ "gfx908:xnack-" "gfx90a:xnack-" "gfx90a:xnack+" ... ]
```

Comment:
There should not be added new instances where packages are being overwritten to null to disable features. This is an anti pattern including the asserts below.

Reply:
I was under the impression it was to not include those packages and make the closure smaller.
If it's not, I'll change it.

Reply:
``gpuTargets`` is however ``null`` for a different reason.
Is there a reason to change it?
Would it be better to change it to something like ``[ ]``?

Reply:
> I was under the impression it was to not include those packages and make the closure smaller.

It is enough to use lib.optional(s) or if else then for that.

> Would it be better to change it to something like `[ ]`?

That depends on how it should be handled. If only the entries in the list matter than having it by default an empty list is better defined. If there is a difference between null and empty list then it maybe shouldn't be changed.

Reply:
There is no discernible difference, so it should be fine.